### PR TITLE
Add virtualbox-ext-pack and some ansible tweaks

### PR DESCRIPTION
**Pull request by dangarthwaite**

### CODE REVIEW

**Comment by dangarthwaite** [comment]
For https://github.com/SteveParson/xubuntu-18.04-dev-box/issues/6

**Comment by SteveParson** [comment]
Thanks for your contribution! I received 3 PRs within one day, all with overlapping commits so it may take me a few days to get to yours. Much appreciated!

File: provisioning/roles/common/tasks/main.yml [hunk 1]
```diff
@@ -24,12 +24,12 @@
...
 
 - name: Create local .bin
-  shell: mkdir ~/.bin
+  file: path=~/.bin state=directory mode=0755
   become: yes
   become_user: "{{ user }}"
 
 - name: Create Desktop folder
-  shell: mkdir ~/Desktop
+  shell: path=~/Desktop state=directory mode=0755
```
Comment (guntbert) [inline provisioning/roles/common/tasks/main.yml]: should be `file:`

Comment (dangarthwaite) [reply provisioning/roles/common/tasks/main.yml]: Added a new commit for this.  Good catch.

**Comment by dangarthwaite** [comment]
@guntbert  Ready to merge.

**Comment by guntbert** [comment]
@dangarthwaite I don't have any privileges here - it was just a drive-by-comment :-))

**Comment by SteveParson** [comment]
Thanks for your contribution!

**Pull request merged**